src/cmd/run.js

```diff
@@ -280,6 +328,7 @@ export default async function run(
         sourceDir,
         artifactsDir,
         addonId,
+        installedExperiments,
```

Comment:
Did you consider making this more abstract in a way that will support both experiments and regular extensions? Maybe a structure like this could accommodate both:
````js
type Extension = {
  addonId: string,
  sourceDir: string,
};

const extensions: Array<Extension> = [];

// Add experiments:
const experiments = await runner.installExperiments(client);
experiments.forEach(
  (experiment) => extensions.push({
    addonId: experiment.addonId, 
    sourceDir: experiment.sourceDir,
  })
);


// Add the extension under test:
const installResult: FirefoxRDPResponseAddon = 
  await runner.installAsTemporaryAddon(client);
extensions.push({addonId: installResult.addon.id, sourceDir});

// Now all functions should be able to work with the array of extension data.
````